src/pages/FarmV3/FarmTableComponents/NoResultsFound.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react'
+import { useDarkMode } from '@/context'
+import { Button, cn } from 'gfx-component-lib'
+import Lottie from 'lottie-react'
+import NoResultFarmdark from '@/animations/NoResultFarmdark.json'
+import NoResultFarmlite from '@/animations/NoResultFarmlite.json'
+
+const NoResultsFound: FC<{ str?: string; subText?: string; requestPool?: boolean }> = ({
+  str,
+  subText,
+  requestPool
+}) => {
+  const { mode } = useDarkMode()
```

Comment:
useDarkMode is kinda strange hook. It writes in LocalStorage variable `darkMode: 'dark'`. And it operates with only two states: `dark` and `lite`(it is clearer to name it light). If there are two states, it is better to return boolean value `isDarkMode`. And many checks in the whole codebase like `mode === dark` will go away